### ISSUE TITLE
[Bug]: Animated gifs not working with next/image on netlify

### ISSUE THREAD

**Issue by danez** [netlify/next-runtime#1841]
### Summary

When I import an gif with next/image the image processing on netlify somehow makes the gif not animate anymore.

### Steps to reproduce

```ts
import Image from 'next/image';

export const whoa = (
  <Image
    src="/mind-blown-shocked.gif"
    alt="whoa"
    width={30}
    height={50}
    style={{ display: 'inline-block' }}
  />
);
```

https://react-docgen.dev/_next/image?url=%2Fmind-blown-shocked.gif&w=64&q=75

### A link to a reproduction repository

https://github.com/reactjs/react-docgen/tree/main/packages/website

### Next Runtime version

4.29.3

### More information about your build

- [ ] I am building using the CLI
- [X] I am building using file-based configuration (`netlify.toml`)

### What OS are you using?

Mac OS

### Your netlify.toml file

<details>
  <summary>`netlify.toml`</summary>

```toml
[build]
command = "pnpm build"
publish = ".next"

[[plugins]]
package = "@netlify/plugin-nextjs"

```

</details>

### Your `next.config.js` file

<details>
  <summary>`next.config.js`</summary>

```toml
const withNextra = require('nextra')({
  theme: 'nextra-theme-docs',
  themeConfig: './theme.config.tsx',
});

module.exports = withNextra();

```

</details>

### Builds logs (or link to your logs)

https://app.netlify.com/sites/react-docgen/deploys/639d0da968a9cd0008809b18

### Function logs

<details>
  <summary>Function logs</summary>

```
empty
```

</details>

### .next JSON files

<details>
  <summary>generated .next JSON files</summary>

```
# Paste file contents here. Please check there isn't any private info in them
# You can either build locally, or download the deploy from Netlify by clicking the arrow next to the deploy time.
```

</details>

**Comment by ascorbic** [comment]
Yeah, the problem here is that it's automatically converting it to webp. I'm not sure what the best workaround is for that (i.e. whether we need to special-case gifs)

**Comment by danez** [comment]
I was also reading #787 which seems to be the same issue and I wonder why it doesn't work, as ipx does seem to have added support for gif: https://github.com/unjs/ipx/blob/main/src/ipx.ts#L125

**Comment by ascorbic** [comment]
It's because we use content negotiation, which means we explicitly set the output format. Otherwise we'd only ever return the response in the same format as the original image, rather than using next-gen formats like webp

**Comment by sarahetter** [comment]
Closing as duplicate https://github.com/netlify/next-runtime/issues/1841
